src/components/CodeOverview/index.tsx

```diff
@@ -53,6 +59,7 @@ export class CodeOverviewBase extends React.Component<Props, State> {
     _window: window,
     createOverviewRef: () => React.createRef<HTMLDivElement>(),
     // This is the padding of the overview container.
+    getCodeLineAnchor: _getCodeLineAnchor,
```

Comment:
We tend to use underscore variables for the prop names, no? (leaving the default imported names unchanged).

Reply:
I hardly get the difference. We use underscore in names of injected props all the time and not exclusively for testing purpose, so it'd be nice to keep things consistent. If you insist on having a non-underscore name as a prop name, then I'd suggest to rename `_getCodeLineAnchor` to `defaultGetCodeLineAnchor`.

Reply:
> We use underscore in names of injected props all the time and not exclusively for testing purpose

Where do we use an underscore prefixed injected prop for something other than testing? That would be confusing and I'd rather not do that anywhere. I guess we adopted the underscore thing from Python since it makes pseudo-private accessors.

> ...then I'd suggest to rename `_getCodeLineAnchor` to `defaultGetCodeLineAnchor`.

I could do that but since we don't do that elsewhere it would introduce an inconsistency. I prefer this style, anyway, as it was the style of `web-ext` 😛 but I don't want to go against the flow.

Reply:
I switched it to a `default*` style but I just made the name more readable as a sentence: `defaultCodeLineAnchorGetter`.